### DISCUSSION TITLE
question PHP

### DISCUSSION THREAD

**Discussion by anhdde** [chamilo/chamilo-lms#5144] [General]
Hello,

I am contacting you on behalf of my IT department. I am a user of the Chamilo platform and I am interested in its future developments. Currently, I understand that Chamilo operates with PHP 7.4. Could you please inform me if there are any plans to update the platform to make it compatible with PHP 8.2?

This update would be beneficial for many reasons, including improved performance and enhanced security.

Thank you in advance for your response.

Best regards,

**Comment by ywarnier** [comment]
Hi @anhdde 

Sure. First off, you *could* actually run Chamilo 1.11.26 on PHP 8.1 (only that one) but it's a bit tricky to do so, and some charts will not be generated anymore (an issue with the charting library we are using). So you're not completely blocked if security is the major issue here. This being said, PHP 7.4 is out of PHP's support lifecycle, but there's no known issue with it either (yet).

Second, we are working (very hard, truth be told) to release Chamilo 2.0 before June this year. Chamilo 2.0 is still in development and I hope we will be able to release the first Alpha in approximately 2 weeks from now, then about another step in our release cycle (alpha, beta, RC, stable) every month. Version 2.0 fully supports (already) PHP 8.1, 8.2 and 8.3, but there are many things that we still want to improve before freezing the db and files structure, which is what holds the release of alpha1.

It is important to note that Chamilo 2.0 does not cover the *full* scope of Chamilo 1.11.26 features, as it was untangled from 1.11.* about 2 years ago and only some of the features since then have been transfered to 2.0, but you'll still get about 95% of the things available in 1.11 (I don't think a lot of people will notice what's missing).

So... still a bit of patience supporting PHP 7.4 and we'll offer a solution together with a new major version, a new design and new features.

**Comment by anhdde** [comment]
Thank you for your time to answer my questions. I really appreciate your help and the clarity you brought to the question.

Sincerely,
